blog/README.md

```diff
@@ -0,0 +1,232 @@
+# Red Hat blog
+
+In this blog we are going to explore the coreos-layering technology in order to
+inject kernel-modules to an Openshift node.
+
+### Prerequisits
+
+* OCP 4.12 or later.
+
+### Background
+
+This time we are going to build a container image that contains a kernel-module
+that we wish to load into the node's kernel.
+
+We will use the [driver-toolkit](https://github.com/openshift/driver-toolkit)
+as a base image in order to build the [simple-kmod](https://github.com/openshift-psap/simple-kmod)
+kernel module and then add the `.ko` to a new layer on top of the REHL-CoreOS
+image that runs on the node.
+
+### Get the correct driver-toolkit image
+
+The driver-toolkit contains the kernel package and headers in order to build
+kernel modules for a specific kernel, therefore, we need to make sure we use
+the correct digest of the driver-toolkit image to fit our specific node.
+
+Let's get the driver-toolkit image for our 4.12.18 cluster
+```
+$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.12.18-x86_64 --image-for=driver-toolkit
```

Comment:
We have an outstanding TODO for adding a proper external pull spec for rhel-coreos, something like:
`quay.io/openshift-release-dev/rhel-coreos:4.13`
that would e.g. float to "latest 4.13" (and we'd have `4.13.7` etc too).
If we go down this path, we clearly want to do the same thing with driver-toolkit.

Reply:
Sounds like a great plane. This will be much easier for customers to consume.
Can you please link the TODO so I can keep an eye on its progress and follow up with DTK?